models/refund_db.go

```diff
@@ -6,6 +6,7 @@ type RefundResourceDB struct {
 	CreatedAt         string `bson:"created_at"`
 	Amount            int    `bson:"amount"`
 	Status            string `bson:"status"`
+	Attempts          int    `bson:"attempts"`
```

Comment:
Does this create an empty `attempts` field upon creation of the refund?
`omitempty` could be used here

Reply:
I was expecting it to take 0 being integer if empty.Anyway added with lots of thanks.